<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="P4ALAP-7172YCJ" ignored="0" name="Sample test 2" tests="3" failures="1" timestamp="16 Feb 2018 15:23:19 GMT" time="48.401" errors="0">
  <testcase name="LoginTest" time="25.038" classname="demo.acn.internal.tests.Sample_Test_2"/>
  <testcase name="LoginTest" time="20.886" classname="demo.acn.internal.tests.Sample_Test_2">
    <failure type="java.lang.AssertionError" message="Login functionality failing. Expected: Login Success but Actual: Login Failure">
      <![CDATA[java.lang.AssertionError: Login functionality failing. Expected: Login Success but Actual: Login Failure
	at demo.acn.internal.utilities.GenericKeywords.reportFailure(GenericKeywords.java:275)
	at demo.acn.internal.utilities.AppKeywords.verifylogin(AppKeywords.java:57)
	at demo.acn.internal.utilities.Keywords.executeKeywords(Keywords.java:58)
	at demo.acn.internal.tests.Sample_Test_2.LoginTest(Sample_Test_2.java:50)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- LoginTest -->
  <testcase name="LoginTest" time="0.007" classname="demo.acn.internal.tests.Sample_Test_2">
    <skipped/>
  </testcase> <!-- LoginTest -->
</testsuite> <!-- Sample test 2 -->
